styles/pronouns.cc/catppuccin.user.css

```diff
@@ -0,0 +1,235 @@
+    --bs-badge-color: @text;
+    --bs-link-color-rgb: #rgbify(@blue) [];
+    --bs-body-color: @text;
+    --bs-body-bg: @crust;
+    --bs-darkk-rgb: #rgbify(@base);
+    --bs-lightt-rgb: #rgbify(@base);
+    --bs-emphasis-color: @accent-color;
+    --bs-emphasis-color-rgb: #rgbify(@accent-color) [];
+    --bs-secondary-color: @subtext0;
+    --bs-secondary-color-rgb: @subtext0;
+    --bs-secondary-bg: @base;
+    --bs-secondary-bg-rgb: #rgbify(@base) [];
+    --bs-tertiary-color: @subtext1;
+    --bs-tertiary-color-rgb: @subtext1;
+    --bs-tertiary-bg: @surface0;
+    --bs-tertiary-bg-rgb: #rgbify(@surface0) [];
+    --bs-primary-text-emphasis: @accent-color;
+    --bs-secondary-text-emphasis: @subtext1;
+    --bs-success-text-emphasis: @green;
+    --bs-info-text-emphasis: @teal;
+    --bs-warning-text-emphasis: @yellow;
+    --bs-danger-text-emphasis: @red;
+    --bs-light-text-emphasis: @subtext0;
+    --bs-primary-bg-subtle: @mantle;
+    --bs-secondary-bg-subtle: @base;
+    --bs-primary-border-subtle: @blue;
+    --bs-secondary-border-subtle: @overlay0;
+    --bs-success-border-subtle: @green;
+    --bs-info-border-subtle: @teal;
+    --bs-warning-border-subtle: @yellow;
+    --bs-danger-border-subtle: @red;
+    --bs-light-border-subtle: @overlay1;
+    --bs-dark-border-subtle: @surface1;
+    --bs-heading-color: inherit;
+    --bs-link-color: @blue;
+    --bs-link-hover-color: @sky;
+
+    --bs-code-color: @pink;
+    --bs-highlight-color: @rosewater;
+    .list-group {
+      --bs-list-group-color: var(--bs-body-color);
+      --bs-list-group-bg: var(--bs-body-bg);
+      --bs-list-group-action-color: var(--bs-secondary-color);
+      --bs-list-group-action-hover-color: var(--bs-emphasis-color);
+      --bs-list-group-action-hover-bg: var(--bs-tertiary-bg);
+      --bs-list-group-action-active-color: var(--bs-body-color);
+      --bs-list-group-action-active-bg: var(--bs-secondary-bg);
+      --bs-list-group-disabled-color: var(--bs-secondary-color);
+      --bs-list-group-disabled-bg: var(--bs-body-bg);
+      --bs-list-group-active-color: @base;
+      --bs-list-group-active-bg: @accent-color;
+      --bs-list-group-active-border-color: @accent-color;
+      border-radius: var(--bs-list-group-border-radius);
+    }
+    // navbar
+    .navbar {
+      --bs-navbar-color: rgba(var(--bs-emphasis-color-rgb), 0.65);
+      --bs-navbar-hover-color: rgba(var(--bs-emphasis-color-rgb), 0.8);
+      --bs-navbar-disabled-color: rgba(var(--bs-emphasis-color-rgb), 0.3);
+      --bs-navbar-active-color: @text !important;
+      --bs-navbar-brand-color: @text !important;
+      --bs-navbar-brand-hover-color: @text !important;
+      --bs-navbar-toggler-border-color: rgba(
+        var(--bs-emphasis-color-rgb),
+        0.15
+      );
+    }
+    #coloredButton(@color, @bg) {
+      --bs-btn-color: @color;
+      --bs-btn-bg: @bg;
+      --bs-btn-border-color: @bg;
+      --bs-btn-hover-color: @color;
+      --bs-btn-hover-bg: fade(@bg, 5%);
```

Comment:
Why is this 5%? It is barely visible.

![CleanShot 2024-10-11 at 13 40 17](https://github.com/user-attachments/assets/2785d04b-9e82-4797-b7e4-31a0d6003572)

Reply:
This doesn't seem fixed? Please don't mark things as resolved if they are not fixed! Even better you can leave them unresolved and I'll resolve them myself if they are fixed.